javalib/src/main/scala/java/util/concurrent/TimeUnit.scala

```diff
@@ -0,0 +1,129 @@
+package java.util.concurrent
+
+// copy&paste from Scala.js
```

Comment:
Lets standardise on `// Ported from XXX.` for this. We should re-add those for all the other things we ported too eventually for consistency.